doc/release/notes.rst

```diff
@@ -23,19 +23,19 @@ Each link leads to the release notes page of the corresponding version:
 
         *   -   2.10
             -   n/a
-            -   :doc:`2.10.0-beta1 <2021-08-releases>`
```

Comment:
this document still exists

Reply:
If there's a page /release/notes, why there's also /release/release-calendar and not /release/calendar?